# get_angle_act_1.msg

string hand_name
int32[] finger_ids       
int32[] angle_values
string[] finger_names     

